test/boulder-tools/build.sh

```diff
@@ -43,15 +43,11 @@ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/insta
 # Note: The version of golang/protobuf is partially tied to the version of grpc
 # used by Boulder overall. Updating it may require updating the grpc version
 # and vice versa.
-GO111MODULE=on go get \
```

Comment:
Did you mean to remove the `GO111MODULE=on` part of this line?

Reply:
Yes. That env var is necessary to make `go get` behave like `go install`; `go install` does the right thing by default.

At least that what I got from the docs.